### PR TITLE
[WNMGDS-2246] Remove old unused focus styles

### PR DESCRIPTION
**Pull request by line47**
## Summary

- There were many variations for focus colors that were old and unused - they have been removed

## How to test

1. See that the following focus variables are no longer in the codebase
  - `focus`, `focus-shadow`, `focus-shadow-inverse`, `focus-shadow-link`, `focus-shadow-link-inverse`, `focus-border-inverse`, `focus-inverse`

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [ ] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [ ] Selected appropriate `Impacts`, multiple can be selected.

### If this is a change to code:

- [ ] Created or updated unit tests to cover any new or modified code
- [ ] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [ ] Checked for spelling and grammatical errors

### CODE REVIEW

**Comment by line47** [comment]
[focus-border-inverse is used here](https://github.com/CMSgov/design-system/pull/2417/files#diff-c0e4a503ab9daaf6604ff7b2da70a82fb6df1c9d6b9b1f5416cd7de4c55c2c84L125) and I'm not sure what this `ShadowTokens` section is or is used for? 

Do some of the focus-related ones need to be removed here? 
@pwolfert @forrestbaer

**Comment by forrestbaer** [comment]
Hey @line47 do you have docker installed locally? Are you able to run `yarn test:browser:update` and check in the visual changes as well for this PR?  Same would go for #2416

**Comment by forrestbaer** [comment]
Oh I see you're out sick today, I will update snapshots in both your branches, no worries feel better!

**Comment by forrestbaer** [comment]
@pwolfert I've added back the `{ delay: 220 }` to the mask interaction test in this PR to resolve that inconsistent `elem.type` which was sometimes only giving 2 characters.  With that delay it seems to work fine. Updated snapshots as well, this should get this PR though CI.

**Comment by pwolfert** [comment]
> [focus-border-inverse is used here](https://github.com/CMSgov/design-system/pull/2417/files#diff-c0e4a503ab9daaf6604ff7b2da70a82fb6df1c9d6b9b1f5416cd7de4c55c2c84L125) and I'm not sure what this `ShadowTokens` section is or is used for?
> 
> Do some of the focus-related ones need to be removed here? @pwolfert @forrestbaer

@line47, are these possibly used in Sketch? If they're not used anywhere, we should be able to remove the entries for them in the shadow tokens as well 👍

File: packages/docs/static/themes/cmsgov-theme.css [hunk 1]
```diff
@@ -137,7 +130,7 @@
 --shadow-focus: inset 0 0 0 1px #262626;
 --shadow-focus-inverse: inset 0 0 0 1px #262626;
 --shadow-focus-link: 0 3px #262626;
---shadow-focus-link-inverse: 0 3px #7c6210;
+--shadow-focus-link-inverse: 0 3px undefined;
```
Comment (forrestbaer) [inline packages/docs/static/themes/cmsgov-theme.css:133]: Seems like we should remove these as well if they're not going to be used?

Comment (pwolfert) [reply packages/docs/static/themes/cmsgov-theme.css:133]: Yeah, I think that's what Scott was saying above, but nobody seems to know where they're used. We should verify that they aren't used to re-create some of the styles in Sketch

**Comment by line47** [comment]
Thanks y'all - still unwell today feel free to take over this PR if you like 😴

File: packages/docs/static/themes/cmsgov-theme.css (hunk 1 above)
Comment (line47) [reply packages/docs/static/themes/cmsgov-theme.css]: Updated and pushed a few commits

**Comment by forrestbaer** [comment]
Hey Scott, I'm checking snapshots for the latest changes, all looks good so far. Could you add Labels and a Milestone for this PR? They should be `Impacts: Core` since this will be effecting all systems and `Type: Changed` there is only one milestone so you can just choose that one. I could do it for you but just wanted to get you used to this workflow. Thanks!

**Comment by forrestbaer** [comment]
All looks good with snapshots, no changes. I will approve when the labels are set!

**Comment by line47** [comment]
@forrestbaer sure thing! I'm a bit unclear on how to use the new labels and the descriptions weren't helpful for me... 🤔 

For example, `Impacts: Core` description is `Impacts the core DS`. It would be more clear for me if the description was `Impacts the entire design system and all themes`

**Comment by forrestbaer** [comment]
@line47 The `imapacts` tags reflect where this item will go in the notes, if you specify more than one it will show up in more than one.  Thanks for noticing the label on changed, will update.

**Review by forrestbaer — APPROVED** [review]
👍🏻

**Pull request merged**